src/shared/parameter_v2/config_definitions/shoot_or_pass_play_config.yaml

```diff
@@ -0,0 +1,23 @@
+- double:
+    name: abs_min_pass_score
+    min: 0.0
+    max: 1.0
+    value: 0.05
+    description: The absolute minimum pass quality we're willing to attempt
+
+- double:
+    name: pass_score_ramp_down_duration
+    min: 0.0
+    max: 5.0
+    value: 2.0
+    description: >-
+      When we're choosing a pass, we start by looking for a pass with a perfect score
+      of 1, and then over time lower the score we're will to accept. This parameter
+      how fast we ramp down to absMinPassScore. This is in seconds.
```

Comment:
Is this referring to the parameter above? If so, update `absMinPassScore` to `abs_min_pass_score`.

Reply:
fixed